static/js/components/CreatePostForm.js

```diff
@@ -132,8 +134,10 @@ export default class CreatePostForm extends React.Component {
             <div className="posting-policy row">
               <span>
                 Please be mindful of{" "}
-                <a href="#">MIT Discussion content policy</a> and practice good
-                online etiquette.
+                <a href={CONTENT_POLICY_URL}>
+                  MIT Discussion content policy
+                </a>{" "}
```

Comment:
should this be a `Link` instead of an anchor tag?

Reply:
Yup, it should be.